src/App.js

```diff
@@ -1,24 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter, Routes, Route } from "react-router-dom";
 
+import "bootstrap/dist/css/bootstrap.min.css";
+
+import HomePage from "./components/HomePage";
```

Comment:
This code should be `import HomePage from "./pages/HomePage";`, so we need to move HomePage in `pages` folder.